SftpLibsshAdapter/SFTPClient.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "DLLImpExp.h"
+#include "SFTPAdapterInterface/ISFTPClientAdapter.h"
+#include "SFTPAdapterInterface/IConnection.h"
+
+namespace systelab { namespace sftp {
+
+	class SFTPLIBSSHADAPTER_API SFTPClient : public ISFTPClientAdapter
+	{
+	public:
+		SFTPClient() = default;
+		~SFTPClient() = default;
+
+		std::unique_ptr<IConnection> connect(const std::string& ip,
+											 unsigned int port,
```

Comment:
when working with ports, unsigned short is enough as the max number of TCP ports is 16 bits (65535)